src/InputItem.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+class InputItem extends Component {
+   
+    createItem = (itemDescription) => {
+       return {id: Math.random().toString(36).substr(2, 9), name: itemDescription, isChecked: false};
+    }
+    
+
+    sendItem = (event) => {
+        event.preventDefault();
+        this.props.itemCallback(this.createItem(event.target[0].value));
```

Comment:
target[0] is not good practice, please use target.elements.inputItem to be specific